lib/wifi_user/use_case/sponsor_users.rb

```diff
@@ -28,14 +28,15 @@ def sanitise_sponsees(contacts)
   end
 
   def invite_sponsees(sponsor, sponsor_address, sponsees)
-    sponsees.each do |sponsee|
+    failed_sponsees = sponsees.reject do |sponsee|
```

Comment:
I'd be happy without assigning a variable here, feels more "ruby". I think it's clear what we're doing without the name? I'm 60-40 though.

`invite_and_return_failed_sponsees` isn't good naming... right?

Reply:
see above, maybe going the hash route is better?

I didn't like `sponsees.reject` on it's own, as it tells us nothing about what's happening

Reply:
Yeah I would rather not use `sponsees.reject` doesn't describe the issue well imo.

Reply:
Better to assign it to a nice variable